File: AD-HOC/2408/Main.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] scores = new int[3];
        for (int i = 0; i < 3; i++)
            scores[i] = scanner.nextInt();
        Arrays.sort(scores);
        System.out.println(scores[1]);
        scanner.close();
    }
}

File: AD-HOC/2408/main.c
#include <stdio.h>

void sort(int *scores) {
    if (scores[0] > scores[2]) _swap(&scores[0], &scores[2]);
    if (scores[0] > scores[1]) _swap(&scores[0], &scores[1]);
    if (scores[1] > scores[2]) _swap(&scores[1], &scores[2]);
}

void _swap(int *a, int *b) {
    int backup = *a;
    *a = *b;
    *b = backup;
}

int main() {
    int scores[3];
    scanf("%d %d %d", &scores[0], &scores[1], &scores[2]);
    sort(scores);
    printf("%d\n", scores[1]);
    return 0;
}
